.github/workflows/ci.yml

```diff
@@ -173,13 +173,27 @@ jobs:
         run: |
           make -C doc html
 
+      - name: Build PDF Documentation
+        run: |
+          sudo apt update
+          sudo apt-get install -y texlive-latex-extra latexmk
+          pip install .[doc]
```

Comment:
```suggestion
```
You do not need to install the pip reqs again. They have already been installed in the previous step